Math/CountingBits.swift

```diff
@@ -0,0 +1,27 @@
+/**
+ * Question Link: https://leetcode.com/problems/counting-bits/
+ * Primary idea:
+ *   bits[i] = bits[i - p] + 1, while p is the largest power of two that smaller than i;
+ *
+ */
+
+class Solution {
+    func countBits(num: Int) -> [Int] {
+        guard num > 0 else {
+            return [0]
+        }
+
+        var bits = Array<Int>(count: num + 1, repeatedValue: 0)
```

Comment:
You can directly use Array(count: num + 1, repeatedValue: 0) or [Int](count: num + 1, repeatedValue: 0)